tests/framework/utils_cpu_templates.py

```diff
@@ -50,22 +50,20 @@ def tmpl_name_to_json(name):
         template_path = Path(
             f"../resources/tests/static_cpu_templates/{name.lower()}.json"
         )
-        return json.loads(template_path.read_text("utf-8"))
+        return {"name": name, "template": json.loads(template_path.read_text("utf-8"))}
```

Comment:
why do we need names?

Reply:
Based on the name of the template I select set of cpu features that should be visible in the guest.

Reply:
Which commit uses the names? Or will it be in a future patch?